Extensions/Physics3Behavior/JsExtension.js

```diff
@@ -0,0 +1,3721 @@
+/**
+ * This is a declaration of an extension for GDevelop 5.
+ *
+ * ℹ️ Run `node import-GDJS-Runtime.js` (in newIDE/app/scripts) if you make any change
+ * to this extension file or to any other *.js file that you reference inside.
+ *
+ * The file must be named "JsExtension.js", otherwise GDevelop won't load it.
+ * ⚠️ If you make a change and the extension is not loaded, open the developer console
+ * and search for any errors.
+ *
+ * More information on https://github.com/4ian/GDevelop/blob/master/newIDE/README-extensions.md
+ */
+module.exports = {
+    createExtension: function(_, gd) {
+      const extension = new gd.PlatformExtension();
+      extension.setExtensionInformation(
+        'Physics3',
+        'Physics Engine 3.0',
+        'Simulate physics',
+        'Florian Rival, Franco Maciel',
+        'MIT'
+      );
+  
+      var physics2Behavior = new gd.BehaviorJsImplementation();
```

Comment:
Update to 3, and see other comments.

Reply:
There are a lot of major fixes that I need to do, I am working on a small issue right now that's why I have not made any recent commits. Although thanks a lot for bringing this typo to my notice @Bouh .